### PR TITLE
feat: add navbar with login button

### PR DESCRIPTION
**Pull request by AdrianValdes**
hi @paschalidi,
I realized this commit was going to get to big if I added the routing functionality. I even had to revert the commit and I stashed just some hunks :grimacing: , first time I do that! Great experience :D. Anyway, I decided to offer this solution. Looking forward to you comments and advice.

### CODE REVIEW

**Comment by AdrianValdes** [comment]
Now I see it just 34 lines, which is maybe too low. But when I add `react-router-dom` gets messy  super fast. Anyhow, little by little I will get a feeling for the right extension for a commit. Trying hard to be nice with my reviewer, thanks dude!

**Comment by paschalidi** [comment]
34 lines isnt too low :) Dont worry about having small PRs. Worry about having too big!

**Comment by paschalidi** [comment]
Nobody will ever tell you this is a very small PRs. I promise you that

File: src/webapp/src/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  width: 1048px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  margin: 13.5px auto;
+`;
+
+export const Navbar = () => (
+  <HeaderWrapper>
```
Comment (paschalidi) [inline src/webapp/src/Navbar.tsx:24]: Not huge fun of the wrapper namings in general. Can we try to call it `HeaderStyles` ?

File: src/webapp/src/App.tsx [hunk 2]
```diff
@@ -3,9 +3,11 @@ import { Button, GlobalStyle } from '@small-ads/ui';
 import { Characters } from './characters/Characters';
 import { Person } from './person/Person';
 import { Auth } from './auth/Auth';
+import { Navbar } from './Navbar';
 
 export const App = () => (
   <>
+      <Navbar />
```
Comment (paschalidi) [inline src/webapp/src/App.tsx:10]:
```suggestion
    <Navbar />
```

**Review by paschalidi — APPROVED** [review]
Looks good. Will approve but good if you can see if my comments make sense to you

File: src/webapp/src/Navbar.tsx (hunk 1 above)
Comment (AdrianValdes) [reply src/webapp/src/Navbar.tsx]: It is done.

**Pull request merged**